recipes/zeromq/all/conanfile.py

```diff
@@ -30,10 +30,6 @@ def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
 
-    def configure(self):
```

Comment:
Why remove?
Create the following conanfile:
```python
from conans import ConanFile
class TttConan(ConanFile):
    name = "ttt"
    version = "0.1"
    options = {
        "shared": [True, False],
        "fPIC": [True, False],\
    }
    default_options = {
        "shared": False,
        "fPIC": True,
    }
    settings = "os", "arch", "compiler", "build_type"
```
Run the following commands on Linux:
```
conan create . ttt/0.1@ -o ttt:shared=True -o ttt:fPIC=False
conan create . ttt/0.1@ -o ttt:shared=True -o ttt:fPIC=True
```
It will create 2 packages:
run
```
tree ~/.conan/data/ttt/0.1/_/_/package
```

Reply:
okay, I just revert this change, take a look again.